lib/cell.rb

```diff
@@ -21,7 +21,46 @@ def fired_upon?
   end
 
   def fire_upon
-    @fired_upon = true
+    case ship
+    when nil
+              #May need to add "Miss" here
```

Comment:
Thinking ahead for 2 potentials:
1 - "You Missed!"
2 - "You've already fired at that spot"

Reply:
Good thinking, made a note to add these